### PR TITLE
Add item

### PR DESCRIPTION
**Pull request by TheDoplarEffect**

### CODE REVIEW

**Comment by TheDoplarEffect** [comment]
Main Additions are in pages/api/addItems.ts file

The point of this endpoint is to add an item to a list (someone has typed in and pressed enter).
First check the name given against the database.
Since there are multiple ways to spell something we are trying to remove  a couple of letters at a time and see if there is a match
When shorter than 3 letters we just put it in the Other list

File: tsconfig.json [hunk 1]
```diff
@@ -1,14 +1,14 @@
...
+    "target": "esnext",
     "lib": ["dom", "dom.iterable", "esnext"],
     "allowJs": true,
     "skipLibCheck": true,
     "strict": true,
     "forceConsistentCasingInFileNames": true,
     "noEmit": true,
     "esModuleInterop": true,
-    "module": "esnext",
+    "module": "commonjs",
```
Comment (NoHara42) [inline tsconfig.json:11]: you cannot parse the entire project as if it were nodejs, as the rest runs on the browser.

If you need to run code on the server, provide a `tsconfig.json` with `"module": "commonjs"` in the same folder as your server files, keeping the root `tsconfig.json` its original value

**Review by NoHara42 — CHANGES_REQUESTED** [review]
reverse merge before PR, also see comments @TheDoplarEffect

File: .env [hunk 2]
```diff
@@ -0,0 +1,13 @@
+# Environment variables declared in this file are automatically made available to Prisma.
```
[the remark this answers is not in the document]
Comment (NoHara42) [reply .env:1]: git rm --cached name_of_file

[the remark this answers is not in the document]
Comment (NoHara42) [reply .env]: file is still committed... @TheDoplarEffect

File: tsconfig.json (hunk 1 above)
Comment (NoHara42) [reply tsconfig.json]: module is still `commonjs` here

**Review by NoHara42 — CHANGES_REQUESTED** [review]
please don't commit .env or lock files, also see unadapted amendments

**Review by NoHara42 — APPROVED** [review]
LGTM!

**Pull request merged**